smoderp2d/time_step.py

```diff
@@ -1,265 +1,354 @@
 # @package smoderp2d.time_step methods to perform
 #  time step, and to store intermediate variables
 
+from itertools import cycle
+from uu import Error
```

Comment:
Is it really necessary to import this special error? This one should be used only for encoding/decoding. I don't see this use in the new code.